src/components/video/Video.styles.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+import { colors } from '../../theme/colors'
+
+export const VideoContainer = styled.div`
+  position: relative;
+  padding-bottom: 56.25%;
+  margin-bottom: 2em;
+
+  iframe {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+  }
```

Comment:
This can be applied to the `iframe` as a `styled-component` rather than relying on inheritance.